crates/nu-cli/src/completions/file_completions.rs

```diff
@@ -157,3 +151,17 @@ pub fn matches(partial: &str, from: &str) -> bool {
     from.to_ascii_lowercase()
         .starts_with(&partial.to_ascii_lowercase())
 }
+
+// escape paths that contains some special characters
+pub fn escape_path_str(path: String) -> String {
+    let mut path = path;
+    if path.contains(' ') || path.contains('\\') || path.contains('"') || path.contains('`') {
+        // Escape characters
+        path = path.replace('"', "\"");
```

Comment:
```suggestion
        path = path.replace('\\', "\\\\");
        path = path.replace('"', "\\"");
```

Reply:
done

Reply:
Oops, `"\\""` was wrong. Thanks for correcting it!